src/main/java/me/ruibin/leto/tasklist/TaskList.java

```diff
@@ -0,0 +1,172 @@
+package me.ruibin.leto.tasklist;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import me.ruibin.leto.storage.Storage;
+import me.ruibin.leto.ui.Ui;
+
+/** Class representing the list of tasks in Leto. */
+public class TaskList {
+    private static final ArrayList<Task> list = new ArrayList<>(100);
+
+    public TaskList() {}
+
+    /** Static method to initialise the task list from csv file. */
+    public static void initFromFile() {
+        Storage.readFile();
+    }
+
+    /** Save task list to csv file. */
+    public static void saveTasks() {
+        Storage.writeFile();
+    }
+
+    /**
+     * Provides a way to add a task to task list.
+     * For example in Storage class readEntry.
+     * */
+    public static void addTaskToList(Task t) {
+        list.add(t);
+    }
+
+    /** In charge of reading inputs and adding the corresponding task to the list */
+    public static void addToList(String inputs) {
```

Comment:
I think you can use overloading here! Since the functionality of adding to list is the same, but the argument types are different it may not be necessary to create another named function.

Reply:
Oh right I forgot I could have done that!